operations/ftdc.go

```diff
@@ -711,3 +713,72 @@ func fromMDB() cli.Command {
 		},
 	}
 }
+
+func toT2() cli.Command {
+	return cli.Command{
+		Name:  "t2",
+		Usage: "write data from genny output file to FTDC",
+		Flags: []cli.Flag{
+			cli.StringFlag{
+				Name:  input,
+				Usage: "source genny output data file",
+			},
+			cli.StringFlag{
+				Name:  output,
+				Usage: "write genny output data in FTDC format to this file (default: stdout)",
+			},
+		},
+		Before: requireFileExists(input, false),
+		Action: func(c *cli.Context) error {
+			inputPath := c.String(input)
+			outputPath := c.String(output)
+
+			ctx, cancel := context.WithCancel(context.Background())
+			defer cancel()
+
+			// Prepare the input
+			//
+			inputFile, err := os.Open(inputPath)
+			if err != nil {
+				return errors.Wrapf(err, "problem opening file '%s'", inputPath)
+			}
+
+			// All genny output files are named using the workload actor and operation.
+			//   i.e., Actor.Operation.ftdc
+			//
+			// We get the actor and operation names from the ftdc filepath for use
+			// in the translation process.
+			//
+			// e.g. A file named InsertRemoveActor.Remove.ftdc will extract
+			// InsertRemoveActor.Remove for the translation process.
+			//
+			fileName := strings.Split(inputPath, "/")
+			actorOperation := strings.Split(fileName[len(fileName)-1], ".ftdc")
+
+			defer func() { grip.Warning(inputFile.Close()) }()
```

Comment:
Nit: move this to line 745, right below the error check after opening the input file.